.circleci/config.yml

```diff
@@ -30,16 +30,34 @@ jobs:
       - store_test_results:
           path: reports/junit
 
-  test-browser:
```

Comment:
Oh god, ignore this diff, it's absolutely unreadable :see_no_evil: 
I reordered the blocks to be in an order consistent with the job list at the bottom and added the node 11 block.